### PR TITLE
Add burn flash sparks to Intake Shooter and Conveyor

### PR DESCRIPTION
**Pull request by balghane**
Add functions for burning the spark max settings for the intake conveyor and shooter sparks

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/shooter/Shooter.java [hunk 1]
```diff
@@ -122,6 +122,15 @@ public boolean initSparks() {
     return errors == 0;
   }
 
+  public void burnFlashSparks() {
+    Timer.delay(0.005);
+    pivotMotor.burnFlash();
+    Timer.delay(0.005);
+    motorA.burnFlash();
+    Timer.delay(0.005);
+    motorB.burnFlash();
```
Comment (allison324) [inline src/main/java/frc/robot/subsystems/shooter/Shooter.java]: make sure to add a third motor

**Review by allison324 — APPROVED** [review]

**Review by tails618 — APPROVED** [review]
Like allison commented, remember to add the third motor once that object is created. Approving here because that is in another PR.

File: src/main/java/frc/robot/subsystems/shooter/Shooter.java (hunk 1 above)
Comment (balghane) [reply src/main/java/frc/robot/subsystems/shooter/Shooter.java]: Thanks, done!

**Pull request merged**